packages/x-components/src/components/suggestions/base-suggestions.vue

```diff
@@ -126,4 +145,33 @@
       }
     }
   ```
+
+  ### Play with props
+
+  In this example, the suggestions has been limited to render a maximum of 3 items.
+
+  _Type “puzzle” or another toy in the input field to try it out!_
+
+  ```vue
+  <template>
+    <div>
+      <SearchInput />
+      <BaseSuggestions :maxItemToRender="3" />
```

Comment:
BaseSuggestions have to receive the suggestions via prop. This example will throuw an error.